lib/rules/jsx-no-constructed-context-values.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
       url: docsUrl('jsx-no-constructed-context-values'),
     },
     messages,
-    schema: {},
+    schema: false,
```

Comment:
This schema wasn't validating or enforcing anything, and ESLint v9 RuleTester now reports it as a possible error (a no-op schema). I'm not sure if `schema: {}` was intentional here, but adding a proper schema now would be a breaking change, so I updated this to `schema: false`. That's an allowed explicit opt-out of schema validation.